### DISCUSSION TITLE
Borders on modules

### DISCUSSION THREAD

**Discussion by ilexite** [polybar/polybar#2681] [Ideas]
I recently designed a bar which featured module borders, I turned to Polybar for this. I later realized that module borders are not possible, only underlines (and overlines), and bar borders too. Here is the bar design, I hope you can maybe implement something like this:
![proto](https://user-images.githubusercontent.com/96395956/161988526-031c3f25-283c-400e-bf22-0fb1d5dfc2eb.png)

**Comment by patrick96** [comment]
You can actually achieve something like this by setting the background to the color you want the border to be, then adding spacing the size of the border and then reset the background color.

Here is a proof of concept:

```dosini
[bar/example]
modules-center = date

font-0 = monospace;3

line-size = 2
height = 24pt

[module/date]
type = internal/date

format-overline = #ff0000
format-underline = #ff0000

format-prefix = %{O2} 
format-prefix-background = #ff0000
format-suffix = %{O2} 
format-suffix-background = #ff0000

time = %H:%M:%S

label = %time%
```

![scrot](https://user-images.githubusercontent.com/2452038/162812507-3a1b5b05-bbf6-4623-a533-987716585095.png)
